doc/openapi.yaml

```diff
@@ -0,0 +1,102 @@
+openapi: 3.0.3
+info:
+  title: Events API
+  version: 1.0.0
+servers:
+  - url: http://localhost:4000
+paths:
+  /api/events:
+    get:
+      summary: Get list of events
+      responses:
+        '200':
+          description: List of events
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/EventsResponse'
+    post:
+      summary: Create a new event
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/EventInput'
+      responses:
+        '201':
```

Comment:
podle kodu muze tvuj server vratit i 400, pokud chybi `title`, to je treba zminit i tady